tools/xdp/s2n-quic-xdp/Cargo.toml

```
@@ -10,11 +10,11 @@ default = ["tokio"]
 bitflags = "2"
 errno = "0.3"
 libc = "0.2"
-s2n-quic-core = { path = "../../../quic/s2n-quic-core", version = "0.19" }
+s2n-quic-core = { path = "../../../quic/s2n-quic-core" }
```

The normal dependency still needs a specific version. We will be publishing this crate as soon as its ready. Can you pin it similar to the other deps?
```suggestion
s2n-quic-core = { version = "=0.20.0", path = "../../../quic/s2n-quic-core" }
```

---

Builds / cargo-watch don't catch that it needs an update-- only the CI caught it when it tried to use the tool. That might be a continued annoyance for future releases.

Why do we need a version if we're using whatever is locally available?